db/seeds.rb

```diff
@@ -5,3 +5,4 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+User.create!(email: 'zander@example.com', password: 'password1234')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.